content/blog/2024/09/camel-quarkus-release-3.15.0/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Camel Quarkus 3.15.0 Released"
+date: 2024-09-23
```

Comment:
So this date would need to be the day the blog post become public.

Reply:
And actually, the public date should be fixed before quarkus platform release, so that the link to the blog post could be mentioned in the quarkus-platform pr. Monday 30th october could be safe date ?

Reply:
> Monday 30th october could be safe date

I set the date to today (25/9).

We should probably rethink the cross-linking of each others blog URLs as the process is a bit flawed IMO.

Reply:
Yes, and maybe not very robust in case of releases delays... need more thinking